src/components/Avatar/Avatar.tsx

```diff
@@ -89,55 +113,43 @@ class Avatar extends UIComponent<any, any> {
   }
 
   renderComponent({ ElementType, classes, rest }) {
-    const { src, alt, name, status } = this.props
+    const { src, alt, name, status, getInitials, size } = this.props
     const { icon = '', color = '' } = Avatar.statusToIcon[status] || {}
```

Comment:
just to not forget - there is a problem here that colors are defined based on status, and thus
- there is no way to customize these colors by theming
- there is no way to decouple Avatar component from the 'presenceStatus' concept - while status aspect of Avatar could be seen as something more general, not about just providing 'presence' status - it could be about providing 'role' status of the person, as a simple example. 

Both these issues should be addressed by means of dedicated PRs - there is a corresponding issue created to track this work:  https://github.com/stardust-ui/react/issues/52

Reply:
Thanks for the review! Will try to create issues for the other things that are not part of this PR